.github/workflows/main.yml

```
@@ -91,8 +91,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: Use oldest RuboCop allowed by gemspec
         run: |
-          sed -nr "s/ *spec.add_dependency 'rubocop', '~> ([0-9\.]+)'/gem 'rubocop', '= \1'/p" \
-            rubocop-rspec.gemspec > Gemfile.local
+          sed -nr "s/ *spec.add_dependency 'rubocop', '~> ([0-9\.]+)'(, '>= ([0-9\.]+)')?/\
+                   gem 'rubocop', '= \3' || '= \1'/p" rubocop-rspec.gemspec > Gemfile.local
```

I think we can ignore the `~>` requirement and only use the `>=` one, right?
```suggestion
          sed -nr "s/ *spec.add_dependency 'rubocop'.*'>= ([0-9\.]+)'/gem 'rubocop', '= \1'/p" \
            rubocop-rspec.gemspec > Gemfile.local
```

---

I considered reverting to `~>` in the future, but this method seems simpler. Thank you!